tools/generate_builtin_frameworks/generate_builtin_frameworks.sh

```diff
@@ -36,11 +36,21 @@ sdk_dirs() {
 
 list_frameworks_for_sdk() {
```

Comment:
One other change is, we no longer sort here since it all gets sorted at the end in `list_frameworks_for_platform` anyway.